apps/vaporgui/RenderHolder.cpp

```diff
@@ -351,8 +351,6 @@ void RenderHolder::_newRendererDialogAccepted()
 	rendererName = uniqueName(rendererName);
 	qname = QString(rendererName.c_str());
 
-	paramsMgr->BeginSaveStateGroup("Create new renderer");
```

Comment:
I haven't tested this, and I'm not sure if it's going to have a consequence.  There is still a call to paramsMgr->EndSaveStateGroup() at line 358 on the current branch, which does not have a BeginSaveStateGroup() associated with it..

Reply:
Good catch!